src/Sylius/Behat/Context/Hook/SpoolContext.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Hook;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Test\Services\EmailCheckerInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+final class SpoolContext implements Context
+{
+    /**
+     * @var string
+     */
+    private $spoolDirectory;
+
+    /**
+     * @var Filesystem
+     */
+    private $fileSystem;
+
+    /**
+     * @param EmailCheckerInterface $emailChecker
+     * @param Filesystem $fileSystem
+     */
+    public function __construct(EmailCheckerInterface $emailChecker, FileSystem $fileSystem)
+    {
+        $this->spoolDirectory = $emailChecker->getSpoolDirectory();
+        $this->fileSystem = $fileSystem;
+    }
+
+    /**
+     * @BeforeScenario @email
```

Comment:
Only if you tag the Feature/Scenario and the Hook is in configuration, but yeah works like a charm 😃